src/chromatix/functional/phase_masks.py

```diff
@@ -153,7 +153,7 @@ def seidel_aberrations(
     l2_sq_grid = X**2 + Y**2
 
     phase *= l2_sq_grid < 1
-    return phase
```

Comment:
Why do we need to add empty dimensions here? This is going to get fixed by `broadcast_2d_to_spatial` in `phase_change` anyway no?

Reply:
We also don't want to assume the batch dimensions